### PR TITLE
Upgraded code-sniffer to 0.17

### PR DESCRIPTION
**Pull request by DmitryLymarenko**

### CODE REVIEW

File: src/Pyz/Zed/DataImport/Business/CombinedProduct/ProductPrice/Writer/CombinedProductPricePropelDataSetWriter.php [hunk 1]
```diff
@@ -16,12 +16,19 @@
 class CombinedProductPricePropelDataSetWriter extends ProductPricePropelDataSetWriter
 {
     protected const COLUMN_ABSTRACT_SKU = CombinedProductPriceHydratorStep::COLUMN_ABSTRACT_SKU;
```
Comment (dmiseev) [inline src/Pyz/Zed/DataImport/Business/CombinedProduct/ProductPrice/Writer/CombinedProductPricePropelDataSetWriter.php:18]: var string

File: src/Pyz/Zed/DataImport/Business/CombinedProduct/ProductStock/Writer/CombinedProductStockPropelDataSetWriter.php [hunk 2]
```diff
@@ -17,7 +17,9 @@
 class CombinedProductStockPropelDataSetWriter extends ProductStockPropelDataSetWriter
 {
     protected const COLUMN_CONCRETE_SKU = CombinedProductStockHydratorStep::COLUMN_CONCRETE_SKU;
+
```
Comment (dmiseev) [inline src/Pyz/Zed/DataImport/Business/CombinedProduct/ProductStock/Writer/CombinedProductStockPropelDataSetWriter.php:20]: var string

File: src/Pyz/Zed/DataImport/Business/Model/ProductPrice/Writer/ProductPricePropelDataSetWriter.php [hunk 3]
```diff
@@ -30,12 +30,19 @@
...
     protected const COLUMN_STORE = ProductPriceHydratorStep::COLUMN_STORE;
+
     protected const COLUMN_CURRENCY = ProductPriceHydratorStep::COLUMN_CURRENCY;
+
     protected const COLUMN_PRICE_GROSS = ProductPriceHydratorStep::COLUMN_PRICE_GROSS;
+
     protected const COLUMN_PRICE_NET = ProductPriceHydratorStep::COLUMN_PRICE_NET;
+
     protected const COLUMN_PRICE_DATA = ProductPriceHydratorStep::COLUMN_PRICE_DATA;
+
```
Comment (dmiseev) [inline src/Pyz/Zed/DataImport/Business/Model/ProductPrice/Writer/ProductPricePropelDataSetWriter.php:45]: same

File: src/Pyz/Zed/DataImport/Business/Model/ProductStock/Writer/ProductStockPropelDataSetWriter.php [hunk 4]
```diff
@@ -35,25 +35,31 @@
 class ProductStockPropelDataSetWriter implements DataSetWriterInterface
 {
     protected const COLUMN_CONCRETE_SKU = ProductStockHydratorStep::COLUMN_CONCRETE_SKU;
+
     protected const COLUMN_IS_BUNDLE = ProductStockHydratorStep::COLUMN_IS_BUNDLE;
+
```
Comment (dmiseev) [inline src/Pyz/Zed/DataImport/Business/Model/ProductStock/Writer/ProductStockPropelDataSetWriter.php:40]: same

File: codeception.ci.functional.yml [hunk 5]
```diff
@@ -2,7 +2,7 @@ namespace: PyzTest
 actor: Tester
 
 include:
-    - tests/PyzTest/*/*
+    - tests/PyzTest/Zed/*
```
Comment (dmiseev) [inline codeception.ci.functional.yml:5]: are you sure?

Comment (DmitryLymarenko) [reply codeception.ci.functional.yml:5]: taken from shop suite
but agree, this skips tests from Service/Client/Glue layers
will revert

**Review by dmiseev — APPROVED** [review]

File: src/Pyz/Zed/DataImport/Business/CombinedProduct/ProductPrice/Writer/CombinedProductPricePropelDataSetWriter.php (hunk 1 above)
Comment (DmitryLymarenko) [reply src/Pyz/Zed/DataImport/Business/CombinedProduct/ProductPrice/Writer/CombinedProductPricePropelDataSetWriter.php:18]: I don't think we will fix such cases
if sniffer is okay with that, then we also good
we have potentially hundreds of such cases and time expenses are too big

Comment (dmiseev) [reply src/Pyz/Zed/DataImport/Business/CombinedProduct/ProductPrice/Writer/CombinedProductPricePropelDataSetWriter.php:18]: discussed with @DmitryLymarenko

**Pull request merged**